_pages/en_US/hiyacfw-setup.md

```diff
@@ -0,0 +1,139 @@
+1. Download the latest Windows version of [hiyaCFW Helper](https://github.com/mondul/HiyaCFW-Helper/releases)
+1. Download the latest windows version of [Just-A-DFC](https://github.com/yourkalamity/just-a-dfc/releases)
+1. Extract the hiyaCFW Helper archive anywhere on your PC
+1. Launch the `HiyaCFW_Helper.exe` file, located in the folder you have just extracted
+1. Leaving the hiyaCFW Helper open, launch the Just-A-DFC `.exe` file you downloaded earlier
+{% endcapture %}
+
+{% capture otherInstructions %}
+### macOS & Linux
+
+1. Download & install the latest version of [Python 3](https://www.python.org/downloads/)
+1. Download the latest python version of [hiyaCFW Helper](https://github.com/mondul/HiyaCFW-Helper/releases)
+1. Download the latest python file of [Just-A-DFC](https://github.com/yourkalamity/just-a-dfc/releases)
+1. Extract the hiyaCFW Helper archive anywhere on your PC
+1. Launch two terminal windows
+   - On macOS, you can launch a terminal window by entering spotlight search (`Command` + `Space Key`) & typing "Terminal"
+   - On Linux, launch a terminal instance via your application launcher
+1. On the first terminal, `cd` to where you extracted the hiyaCFW Helper and type `./HiyaCFW_Helper.py`
+1. On the second terminal, `cd` to where you downloaded `just-a-DFC.py` and type `./just-a-DFC.py`
+{% endcapture %}
+
+<div id="windowsInstructions" class="blanktabcontent">{{ windowsInstructions | markdownify }}</div>
+<div id="otherInstructions" class="blanktabcontent">{{ otherInstructions | markdownify }}</div>
+
+<script>
+	let tabcontent = document.getElementsByClassName("blanktabcontent");
+	let tablinks = document.getElementsByClassName("tablinks");
+
+	function openTab(evt, tabName) {
+		let element;
+
+		for (element of tabcontent) {
+			element.style.display = "none";
+		}
+
+		for (element of tablinks) {
+			element.className = element.className.replace("btn--primary", "btn--info");
+			if (!element.className.includes('btn--info'))
+				element.className += " btn--info";
+		}
+
+		document.getElementById(tabName).style.display = "block";
+		evt.currentTarget.className = evt.currentTarget.className.replace("btn--info", "btn--primary");
+	}
+
+	// Remove links from tab buttons
+	for(a of tablinks) {
+		a.href = "javascript:void(0);";
+	}
+
+	// Open the tab for the current OS
+	if(navigator.platform.includes("Win"))
+		document.getElementById("windows").click();
+	else
+		document.getElementById("other").click();
+</script>
+
+## Section II - Adding hiyaCFW files to your SD card
+![](https://image.ibb.co/hhzKRL/Screen-Shot-2018-10-18-at-16-30-18.png)
+
+1. Switch over to the hiyaCFW window
+1. Click the `...` button in the `NAND file with No$GBA footer` box
+1. Navigate to your NAND backup, and click `Open`
+1. Press `Start`
+1. In the new pop-up window, navigate to the root of your SD card, and press `OK`.
+   - The process may take several minutes
+1. When the application says "Done", close out of hiyaCFW Helper
```

Comment:
The hiya helper is a bit unstable going directly to the SD card, so its better to run it to an empty folder then copy to root
```suggestion
1. In the new pop-up window, create a new empty folder, select that, and press `OK`.
   - The process may take several minutes
1. When the application says "Done", close out of hiyaCFW Helper
1. Copy the contents of the new folder you made to the root of your SD card
```

Reply:
Hmm, I'd rather fix this on hiyaCFW helper's side. I'll see what I can do.